src/shared/route-urls.ts

```diff
@@ -72,11 +72,13 @@ export enum RouteUrls {
   SendOrdinalInscriptionError = 'error',
 
   // Swap routes
-  Swap = '/swap/:base/:quote?',
+  SwapBitcoinAsset = '/swap/bitcoin/:base/:quote?',
```

Comment:
```suggestion
  SwapBitcoinAsset = '/swap/{chain}/:base/:quote?',
```

Not necessary, but if you did wanna keep this less repetitive, can still replace the string when declaring the routes in the route gen fn

Reply:
The reason I ended up not doing this is bc I can replace it when gen in the routes, but then every time I need to use it I also have to do a replace or export a shared const with the replace so we can dynamically change the :base and :quote. When I had it that way initially it felt less clean, but I can switch it back if you think better? I was just exporting then a group of routes with the replace 'SwapRouteUrls'.

Reply:
Maybe then it makes sense just to keep it dynamic as :chain?

Reply:
Happy to leave it as it it. Definitely not worth returning to dynamic routes.

It's a slight reduction in repetition to use "templated routes" (I'll call them), but needed, and indeed you'd need to call `.replace('{chain}', chain)` in many places/

Reply:
I think the templating is better for scaling so will change back this morn.

Reply:
@kyranjamie think this is good now? I created a `constructSwapRoute` helper that I hope simplifies the need to use the replace in several places.

Reply:
yeah looks great, like the helper fn

Reply:
When you get a chance, can you approve so I can merge it?